pyanaconda/modules/payloads/payload/rpm_ostree/installation.py

```diff
@@ -628,10 +628,17 @@ def run(self):
             if not self._data.signature_verification_enabled:
                 args.append("--no-signature-verification")
 
-            safe_exec_with_redirect(
-                "ostree",
-                args
-            )
+            # TODO refactor things so we can log stdout to the program log or journal.
+            # For now we just use anaconda's stdout.
```

Comment:
If you can make it log to Journal, it should be captured even now when something goes wrong - Journal is the de-facto primary log sink for Anaconda these days & what we gather and ask users to attach to bugs. :)

Sure, it would be better if Anaconda got distinct event/callback when something happens, but just making sure everything ends up in Journal would help already. :)